bi_sql_editor/models/bi_sql_view_field.py

```diff
@@ -168,6 +168,8 @@ def create(self, vals):
             many2one_model_id = (
                 self.env["ir.model"].search([("model", "=", model_name)]).id
             )
+            if not many2one_model_id:
+                raise UserError(_("There is no relation matching for the field %s", field_without_prefix))
```

Comment:
I'm wondering if it shouldn't be best to show a popup

Reply:
If it's done in the interface, it will popup a window showing the error message no ?